client/src/components/LayerManager.vue

```diff
@@ -340,6 +372,21 @@ export default defineComponent({
       { deep: true },
     );
 
+    watch(attributes, () => {
+      updateAttributes();
+      updateLayers(
+        frameNumberRef.value,
+        editingModeRef.value,
+        selectedTrackIdRef.value,
+        multiSeletListRef.value,
+        enabledTracksRef.value,
+        visibleModesRef.value,
+        selectedKeyRef.value,
+        props.colorBy,
+      );
+    });
```

Comment:
I don't think we need to watch it deeply because our SetAttribute function does a vue set/vue delete on the values that should trigger reactivity on the array.
https://github.com/Kitware/dive/blob/main/client/src/use/useAttributes.ts#L130-L147
This allows us to be more decisive about when attributes update vs. updating anytime a deeply nested property is modified.  So in attribute editor, you may see that I trigger an update to the attribute a bunch and that's because I want to update the styling in real time for moving attributes.  Let me know if this 


This leads me to think that the dialog for attribute editing should be moved to left/right/bottom of the screen to make the render area a bit easier to see.